ConfigVersionControl/git_version_control.py

```diff
@@ -255,3 +262,26 @@ def add_all_edited_files(self):
         Does a 'git add -u' which adds all edited files.
         """
         self.repo.git.add(u=True)
+
+    def _needs_adding(self, path):
+        """
+        Check if file or directory really needs to be added to git
+        """
+        # we need to make sure paths we compare are consistent in both case and path separators
+        # also as git returns changed/untracked files and we may get passed a directory name, we need
+        # to use "startswith" on the directory prefix later so also make sure it ends with a separator
+        isdir = os.path.isdir(path)
+        relpath = os.path.normcase(os.path.normpath(os.path.relpath(path, str(self.repo.working_tree_dir))))
+        if isdir and not relpath.endswith(os.sep):
+            relpath = relpath + os.sep
+        c1 = [ item.a_path for item in self.repo.index.diff(None) ]
```

Comment:
What does `c1` mean? Could you name this variable something a bit more informative?